vassal-app/pom.xml

```diff
@@ -401,6 +402,7 @@
                 <artifactId>clirr-maven-plugin</artifactId>
                 <configuration>
                     <ignoredDifferencesFile>src/test/resources/clirr-ignored-differences.xml</ignoredDifferencesFile>
+                    <comparisonVersion>${clirr.nofork.comparison.version}</comparisonVersion>
```

Comment:
I've made this change because it is pointless doing clirr nofork comparisons between the betas, which is happening now. The default seems to be the last release which includes betas.

Just means that <clirr.nofork.comparison.version>will need updating for each real release.

I also stripped the majority of the difference rules out of the ignore file as they are no longer relevant.